src/richie/apps/courses/defaults.py

```diff
@@ -242,7 +242,10 @@
 COURSE_GLIMPSE_VARIANT_CHOICES = getattr(
     settings,
     "RICHIE_COURSE_GLIMPSE_VARIANT_CHOICES",
-    [(None, _("Inherit")), ("glimpse", _("Default")), ("small", "small")],
+    [
+        (None, _("Inherit")),
+        ("large", _("Large")),
```

Comment:
you need to add a migration because these choices are in database for someone who did not tweak the setting... 
I would say you also need to keep the "small" option. So the migration could do:
glimpse -> None
small -> small
large (new)

Reply:
Small variant do currently nothing.

Reply:
then ok to remove it with a migration setting the value to "None"... because someone might have it as value in their database.

Reply:
Finally I kept the small variant and its value is 'glimpse' to not display something broken if end user selects this value. Furthermore since I add a `large` variant, `small` variant seems logic.

Reply:
Finally I kept the small variant and its value is 'glimpse' to not display something broken if end user selects this value. Furthermore since I add a `large` variant, `small` variant seems logic.